app/boards/shields/nice_view/nice_view.overlay

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+&nice_view_spi {
```

Comment:
Should we provide this in `shields/nice_view/boards/*.overlay` files for common/in-tree boards?